pycode/memilio-surrogatemodel/memilio/surrogatemodel/ode_secir_simple/model.py

```diff
@@ -156,21 +164,60 @@ def get_test_statistic(test_inputs, test_labels, model):
     test_labels = np.array(test_labels)
 
     diff = pred - test_labels
-    anteil = (abs(diff))/abs(test_labels)
-
-    anteil_6 = anteil.transpose(2, 0, 1).reshape(6, -1)
-    df = pd.DataFrame(data=anteil_6)
-    df = df.transpose()
-
+    relative_err = (abs(diff))/abs(test_labels)
+    # reshape [batch, time, features] -> [features, time * batch]
+    relative_err_transformed = relative_err.transpose(2, 0, 1).reshape(8, -1)
+    relative_err_means_percentage = relative_err_transformed.mean(axis=1) * 100
     mean_percentage = pd.DataFrame(
-        data=(df.mean().values) * 100,
-        index=['Exposed', 'Carrier', 'Infected', 'Hospitalized', 'ICU',
-               'Dead'],
+        data=relative_err_means_percentage,
+        index=['Susceptible', 'Exposed', 'InfectedNoSymptoms',
+               'InfectedSymptoms', 'InfectedSevere', 'InfectedCritical',
+               'Recovered', 'Dead'],
```

Comment:
```suggestion
        index=[str(compartment).split('.')[1] for compartment in InfectionState.values()],
```

Please try to always access variable names instead of rewriting strings.